work7/task7_2.py

```diff
@@ -0,0 +1,11 @@
+def fun(variable):
+    letters = ['a', 'e', 'i', 'o', 'u']
+    if (variable in letters):
+        return True
+    else:
+        return False
+sequence = ['g', 'e', 'e', 'j', 'k', 's', 'p', 'r']
+filtered = filter(fun, sequence)
```

Comment:
looks like you use default filter. You need t implement your own one

Reply:
fix